<launch>
<arg name="forest_path" default="$(find rwth_upper_body_skeleton_detector)/Forest/" />
<node
pkg = "rwth_upper_body_skeleton_client"
type = "rwth_upper_body_skeleton_client_node"
name = "upper_body_skeleton_client"
output = "screen"
/>
<node
pkg = "rwth_upper_body_skeleton_detector"
type = "rwth_upper_body_skeleton_detector"
name = "upper_body_skeleton_server"
output = "screen">
<param name="forest_path" value="$(arg forest_path)" type="string"/>
</node>
</launch>
